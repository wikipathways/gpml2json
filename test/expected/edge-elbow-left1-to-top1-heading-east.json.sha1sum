0ac05d832cc8c14ccfeec0963e712e5f0c3c13fb ?-
