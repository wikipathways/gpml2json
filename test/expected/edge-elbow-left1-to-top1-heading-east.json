{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["ebee3","e691a","a00ff"],"drawAs":"rect","gpmlElementName":"Pathway","height":196.32142416449818,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":395.65483553329415,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/5d272e0b91999dedd86ce099038cc8c563cba4b0e821e3aa47fe00edce3333e5"},"entityMap":{"e691a":{"id":"e691a","textContent":"0017","wpType":"GeneProduct","x":217.47301735147596,"y":82.40101539373862,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32960,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"a00ff":{"id":"a00ff","textContent":"GeneProduct","wpType":"GeneProduct","x":86.73400114925414,"y":119.9577878008619,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32961,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ebee3":{"id":"ebee3","drawAs":"ElbowLine","borderWidth":1,"zIndex":32959,"points":[{"isAttachedTo":"a00ff","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":86.73400114925414,"y":129.9577878008619,"orientation":[-1,0]},{"x":66.73400114925414,"y":96.17940159730026},{"y":62.40101539373862,"x":162.10350925036505},{"isAttachedTo":"e691a","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":257.47301735147596,"y":82.40101539373862,"orientation":[0,1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["a00ff","e691a"]}}}
